src/app/[locale]/(pages)/ripristino-accesso/errore/page.tsx

```diff
@@ -3,13 +3,23 @@
 import { Grid } from '@mui/material';
 import { IllusError } from '@pagopa/mui-italia';
 import { useTranslations } from 'next-intl';
+import { useEffect } from 'react';
 import { FeedbackMessage } from '../../../_component/feedbackMessage/feedbackMessage';
 import { commonBackground } from '../../../_utils/styles';
 import { ROUTES } from '@/app/[locale]/_utils/routes';
+import { trackEvent } from '@/app/[locale]/_utils/mixpanel';
 
 const LoginKo = (): React.ReactElement => {
   const t = useTranslations('ioesco');
 
+  useEffect(() => {
+    trackEvent('IO_PROFILE_UNLOCK_ACCESS_ERROR', { event_category: 'KO', reason: 'UNLOCK_ERROR' });
```

Comment:
The 'reason' field lacks valuable information as the event name already sufficiently explains it.